__tests__/index.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/healthcheck", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/healthcheck").expect(200);
+  });
+});
+
+describe("GET /api/topics", () => {
+  test("return a status code of 200 and response body with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const allTopics = response.body;
```

Comment:
this should be on an appropriate key something like body.topics, this will need to be implemented in the controller